Shopify/ruby-lsp
# RuboCop Code Action "Disable for this line"

### Motivation

Adds a code action that disables the RuboCop rule for a line:

https://github.com/Shopify/ruby-lsp/assets/71686/72b2f0f0-4aac-49d1-a78e-509e740625b7

fixes #1099.

### Automated Tests

See notes inline.

### Manual Tests

1. Check out this branch in vscode
2. Restart the LSP server
3. Edit `.rubocop.yml` to stop excluding `"test/expectations/**/*"`
4. Edit `.vscode/settings.json` and disable `editor.formatOnSave` for `[ruby]
5. Open the `test/expectations/code_actions_formatting` directory.
6. Open files in there and run the "Disable X" code actions on the offenses.
